animation_nodes/ui/node_menu.py

```diff
@@ -344,6 +344,8 @@ def draw(self, context):
         insertNode(layout, "an_ArmatureInfoNode", "Armature Info")
         layout.menu("AN_MT_object_utils_menu", text = "Utils")
         layout.separator()
+        insertNode(layout, "an_ObjectViewportColorNode", "Viewport Color")
```

Comment:
Rename the node to `an_ObjectColorOutputNode` and move it under Data Path Output node in the menu.

Reply:
Omar, I think _Object Viewport Color_ name is more convenient and has a similar meaning as we have in the blender,
![Screenshot from 2020-02-21 15-43-30](https://user-images.githubusercontent.com/30294746/75025361-0fe04a00-54c1-11ea-94af-654f6bd3ecca.png)

Reply:
But it is not really a viewport color. It is just a color that can be used for viewport rendering.

Reply:
Okay.